src/server/apsystem.ts

```diff
@@ -488,16 +493,25 @@ export default class ActivityPubSystem {
 
     const followers = await actorStore.followers.list()
 
-    const items = await Promise.all(
-      followers.map(async (mention) => await this.mentionToActor(mention))
-    )
+    const items = (await Promise.all(
+      followers.map(async (mention) => {
+        try {
+          const url = await this.mentionToActor(mention)
+          return url
+        } catch {
+          // If we can't resolve them just don't show them
+          return ''
+        }
+      })
+      // Filter out failed loads
+    )).filter((item) => item.length !== 0)
 
     return {
       '@context': 'https://www.w3.org/ns/activitystreams',
       type: 'OrderedCollection',
       id: followersURL,
       items,
-      totalItems: items.length
+      totalItems: followers.length
```

Comment:
won't this produce innacurate counts?

Reply:
I think Mastodon does a similar thing where it shows the total count the user reports but doesn't ingest all the followers

Reply:
As long as we document it's the currently active followers list I'm ok with this number